### DISCUSSION TITLE
BUG: Diff mode no longer shows the changes made inside each page

### DISCUSSION THREAD

**Discussion by dfinster** [GitbookIO/community#279] [0. General bugs and questions]
We've noticed that Diff mode now only shows which pages have changed, but no longer shows the changes made within each page. We consider this a major bug and a showstopper for our workflow. Please advise.

> I emailed support@gitbook.com, but posting here also to see if anyone else has noticed this.

**Comment by valentin0h** [comment]
Thank you for reporting this @dfinster. We are aware of the issue and will ship a fix very soon. Sorry for the inconvenience!

**Comment by taszap** [comment]
Just following up on this, we confirmed via the email but the issue should be fixed!
